common/views/components/CardGrid/CardGrid.js

```diff
@@ -39,7 +39,7 @@ const CardGrid = ({
         <div className="css-grid">
           {items.map((item, i) => (
             <div
-              key={item.id}
+              key={i}
```

Comment:
Why not the ID?

Reply:
because we duplicate events if they occur in more than one month

Reply:
Could use `month-id` or something to make it more unique?